batch/batch/driver/instance.py

```diff
@@ -200,6 +200,9 @@ async def make_request():
                 await self.client_session.post(
                     f'http://{self.ip_address}:5000/api/v1alpha/kill', timeout=aiohttp.ClientTimeout(total=30)
                 )
+            except ConnectionRefusedError:
+                await self.deactivate('killed')
```

Comment:
say more?

Reply:
I noticed before I got the workers online the driver got into a loop treating a connection refused as a transient error. I think this was before we introduced limited retries though, so I'll take this out.